Proiect/Proiect.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Fix] acest fisier, cat si folderele .idea si out trebuie adaugate in .gitignore, ele sunt fisiere de configurare locala de intellij si clase compilate local de java. nu au ce cauta in proiect